presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TestExpressionVerifier.java

```diff
@@ -41,6 +52,12 @@ public void test()
         assertTrue(verifier.process(actual, expression("NOT(X = 3 AND Y = 3 AND X < 10)")));
         assertThrows(() -> verifier.process(actual, expression("NOT(X = 3 AND Y = 3 AND Z < 10)")));
         assertFalse(verifier.process(actual, expression("NOT(X = 3 AND X = 3 AND X < 10)")));
+
+        RowExpressionVerifier rowExpressionVerifier = new RowExpressionVerifier(symbolAliases, metadata, TEST_SESSION);
+
+        assertTrue(rowExpressionVerifier.process(expression("NOT(X = 3 AND Y = 3 AND X < 10)"), translate(actual)));
```

Comment:
Shouldn't this have the reverse symbol alias as the ExpressionVerifier?

Reply:
As discussed offline, I will flip the expect and actual argument names